calico_node/Makefile

```diff
@@ -469,6 +472,64 @@ release: clean
 	@echo "docker push quay.io/$(NODE_CONTAINER_NAME):latest"
 	@echo "See RELEASING.md for detailed instructions."
 
+
+RELEASE_DIR:=release-$(CALICO_VER)
+RELEASE_DIR_K8S_MANIFESTS:=$(RELEASE_DIR)/k8s-manifests
+MANIFEST_SRC := ../_site/$(RELEASE_STREAM)/getting-started/kubernetes/installation
+
+## Create an archive that contains a complete "Calico" release
+release-image: $(RELEASE_DIR).tgz
```

Comment:
minor nit, but "image" sounds a lot like a docker image to me.  Might change to "archive" or something.